frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -7,9 +8,13 @@
 
 @RestController
 public class CarController {
-    Map<Integer, Car> map = new HashMap<>();
+   private Map<Integer, Car> map = new HashMap<>();
```

Comment:
Since you are now injecting the map into the constructor you don't need this `new HashMap<>`